### PR TITLE
Mention the `kiota-java-extra` repository

### PR DESCRIPTION
**Pull request by andreaTP**
It's finally more official, so worth adding it to the Readme 🙂

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,6 +65,12 @@ In `pom.xml` in the `dependencies` section:
     </dependency>
 ```
 
+## Extra
+
+[RedHat](https://www.redhat.com/) is maintaining extra utilities for Java projects based on Kiota:
+
+https://github.com/redhat-developer/kiota-java-extra
```
Comment (baywet) [inline README.md:72]:
```suggestion
Third party tools and extensions is maintaining:

- [Extra utilities for Java projects based](https://github.com/redhat-developer/kiota-java-extra) by [RedHat](https://www.redhat.com/). Kiota Maven plugin to ease the usage of the Kiota CLI from Maven projects.
```

**Review by baywet — CHANGES_REQUESTED** [review]
That's awesome news! What do you think of this format instead?

File: README.md (hunk 1 above)
Comment (andreaTP) [reply README.md]: works for me 👍  thanks!

**Review by baywet — APPROVED** [review]

**Comment by baywet** [comment]
if you want we could start a PR for the docs site (microsoft.github.io/kiota) on the main repo as well. We should be to add:
- this
- [that](https://marketplace.visualstudio.com/items?itemName=ChristianResmaHelle.ApiClientCodeGenerator2022)
- and the vs code extension in the future.

Don't hesitate to get started on the PR, adding a new page under the welcome to kitoa section should do it for now.

**Comment by andreaTP** [comment]
Nice! I think I'll wait to have published a first "official" release of the RH Java SDKs, but it might happen pretty soon 🙂 👍

**Pull request merged**